lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -3032,7 +3040,7 @@ namespace Js
                             }
                             else
                             {
-                                SetArrayLikeObjects(pDestObj, idxDest, subItem);
+                                ThrowErrorOnFailure(SetArrayLikeObjects(pDestObj, idxDest, subItem), scriptContext, idxSubItem);
```

Comment:
>idxSubItem [](start = 116, length = 10)

Should this be idxDest? if not can we put a comment explaining why its idxSubItem as we are setting the element at idxDest.